client/modules/Post/PostActions.js

```diff
@@ -1,60 +1,97 @@
-import callApi from '../../util/apiCaller';
+import callApi from "../../util/apiCaller";
 
-// Export Constants
-export const ADD_POST = 'ADD_POST';
-export const ADD_POSTS = 'ADD_POSTS';
-export const DELETE_POST = 'DELETE_POST';
+export const ADD_POST = "ADD_POST";
+export const ADD_POSTS = "ADD_POSTS";
+export const DELETE_POST = "DELETE_POST";
+export const EDIT_POST = "EDIT_POST";
+export const LIKE_POST = "LIKE_POST";
+export const DISLIKE_POST = "DISLIKE_POST";
 
-// Export Actions
 export function addPost(post) {
   return {
     type: ADD_POST,
-    post,
+    post
   };
 }
 
 export function addPostRequest(post) {
-  return (dispatch) => {
-    return callApi('posts', 'post', {
+  return dispatch => {
+    return callApi("posts", "post", {
       post: {
         name: post.name,
         title: post.title,
-        content: post.content,
-      },
+        content: post.content
+      }
     }).then(res => dispatch(addPost(res.post)));
   };
 }
 
 export function addPosts(posts) {
   return {
     type: ADD_POSTS,
-    posts,
+    posts
   };
 }
 
 export function fetchPosts() {
-  return (dispatch) => {
-    return callApi('posts').then(res => {
+  return dispatch => {
+    return callApi("posts").then(res => {
       dispatch(addPosts(res.posts));
     });
   };
 }
 
 export function fetchPost(cuid) {
-  return (dispatch) => {
+  return dispatch => {
```

Comment:
why you removed `()`?

Reply:
Prettier deleted parentheses :( I have to check it